src/extension.ts

```diff
@@ -163,6 +172,41 @@ export async function activate(context: vscode.ExtensionContext) {
 		}),
 	);
 
+	context.subscriptions.push(
+		vscode.commands.registerCommand('intuita.signIn', () => {
+			const searchParams = new URLSearchParams();
+
+			searchParams.set(
+				SEARCH_PARAMS_KEYS.COMMAND,
+				'accessTokenRequested',
+			);
+
+			const url = new URL('https://codemod.studio');
+			url.search = searchParams.toString();
+
+			vscode.commands.executeCommand('intuita.redirect', url);
+		}),
+	);
+
+	context.subscriptions.push(
+		vscode.commands.registerCommand('intuita.signedIn', async () => {
```

Comment:
oh I see, I would call it with a verb, e.g. `intuita.handleSignedInUser` or something similar that conveys what kind of action we want to perform

Reply:
agree, addressed